web-client/public/js/graph.js

```diff
@@ -70,7 +70,7 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
 
   // normalization all weights b/w 2-14
     var normMax = +$("#normalization-max").val();
-    var totalScale = d3.scale.linear()
+    var totalScale = d3.scaleLinear()
     .domain([0, normalization && normMax > 0 ? normMax : d3.max(allWeights)])
     .range([2, 14])
     .clamp(true);
```

Comment:
Our linter is OK with this? I would have expected `.domain`, `.range`, and `.clamp` to be indented more deeply.

Reply:
Yes, linter is ok with this. @yshin4 and I do think it looks better if they are indented so we went ahead and put the indentation in (linter is ok with this as well)